frontend/simple/views/Contributions.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <main>
+    <div class="section is-hero">
+      <i18n tag="h1" class="title is-1 is-marginless">
+        Contributions
+      </i18n>
+      <i18n tag="p">See everyone’s contributions to the group and manage yours.</i18n>
+    </div>
+
+    <div class="section columns is-desktop is-multiline c-grid">
+      <section class="column">
+        <i18n tag="h2" class="title is-3">Receiving</i18n>
+
+        <ul class="c-ul">
+          <contribution
+            v-for="contribution, index in receiving"
+            :variant="isMonetary(contribution.type) ? 'editable' : undefined"
+            v-on="isMonetary(contribution.type) ? { click: showReceivingSettings } : {}"
+          >
+            <strong v-if="isMonetary(contribution.type)">{{textGivingMonetary(index)}}</strong>
+            <strong v-else>{{contribution.what}}</strong>
+            <i18n>from</i18n>
```

Comment:
Similar to the [comment below](
https://github.com/okTurtles/group-income-simple/pull/480/files#r215454403), the problem with chopping up strings like this (outside of `<i18n>` tags), is that some languages may reverse the order of words.

In other words, here we have a sentence that looks like this:

```
Cooking from Lilia Bouvet
```

But another language might write that like so:

```
Bouvet Lilia Cooking from
```

Or something else entirely crazy.

This is why you need to go through the entire PR and look at all areas like this, and ensure that all *sentences* are enclosed entirely within a single `<i18n>` tag, and use the `:args=` binding to pass in variables.

Reply:
It makes sense, I'll look into that later.

Reply:
Here we're dealing with text within html tags and part of it has a button that is a tooltip... Kinda tricky. @taoeffect how do you suggest to implement the translation here?